src/main/java/seedu/duke/common/InfoMessages.java

```diff
@@ -20,6 +20,12 @@ public enum InfoMessages {
     INFO_LIST_UNFILTERED("There are no transaction records that match your search expression."),
     INFO_STATS_EMPTY("There are no statistics available yet for the given statistics type."),
     INFO_STATS_CATEGORIES("Here are the total savings for each category:"),
+    INFO_STATS_MONTHS("Here is the expenditure summary for each month:"),
```

Comment:
Descriptions looks good. 
Maybe could possibly change the naming to something more explicit such as "INFO_STATS_MONTHS_COMMENT_HALF_INCOME_SAVED" instead of "INFO_STATS_MONTHS_COMMENT_THREE".

Reply:
Yes, thank you for this feedback. They have been renamed as in this PR: https://github.com/AY2223S1-CS2113-W12-2/tp/pull/134